tests/upgrades/test_user.py

```diff
@@ -45,17 +54,51 @@ def test_pre_create_sshkey_in_existing_user(self):
         :expectedresults: The user should be created successfully
         """
 
+        login_name = gen_string('alpha')
+
+        # Creating User
+        entities.User(login=login_name).create()
+
+        # Verify if created user exists
+        check_user = entities.User().search(query={'search': 'login=' + str(login_name)})
+        assert len(check_user) == 1
+        assert check_user[0].login == login_name
+
+        # Adding the User details to dict
+        create_dict(
+            {self.__class__.__name__: {'user_login': check_user[0].login,
+                                       'user_id': check_user[0].id}}
```

Comment:
Not required to pass this on to post, just login name is fine. You will get everything using login name.